File: withgoogle/CodeJam/2019/Qualification/Cryptopangrams/BigIntegerGCD/test/SolutionTests.java
import static org.junit.Assert.*;

import java.io.*;

import org.junit.*;

public class SolutionTests {

    @Test
    public void test0() throws FileNotFoundException {
        InputReader in = new InputReader(new FileInputStream("test_data/0.in"));
        Solution solution = new Solution(in);
        String[] result = solution.solve();
        assertArrayEquals(new String[] {
            "Case #1: CJQUIZKNOWBEVYOFDPFLUXALGORITHMS",
            "Case #2: SUBDERMATOGLYPHICFJKNQVWXZ"
        }, result);
    }

    @Test
    public void test1() throws FileNotFoundException {
        InputReader in = new InputReader(new FileInputStream("test_data/1.in"));
        Solution solution = new Solution(in);
        String[] result = solution.solve();
        assertArrayEquals(new String[] {
            "Case #1: BCJQUIZKNOWBEVYOFDPFLUXALGORITHMS",
            "Case #2: CBCJQUIZKNOWBEVYOFDPFLUXALGORITHMS",
            "Case #3: ACBCJQUIZKNOWBEVYOFDPFLUXALGORITHMS"
        }, result);
    }

}

File: withgoogle/KickStart/2019/Round B/Diverse Subarray/AdHocSearch/Solution.Tests/SolutionTests.cs
using System;
using System.IO;
using NUnit.Framework;

public class SolutionTests {

	[Test]
	public void Test0() {
		Solution solution = new Solution(new InputReader(new StreamReader("../../../../test_data/0.in")));
		string[] result = solution.Solve();
		Assert.AreEqual(new string[] {
			"Case #1: 4",
			"Case #2: 6",
			"Case #3: 4",
			"Case #4: 6"
		}, result);
	}

}
